app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/Constants.kt

```diff
@@ -0,0 +1,11 @@
+package au.com.gridstone.trainingkotlin
+
+object Constants {
+  const val MY_TAG = "MY_LOG_TAG"
```

Comment:
is this used?

Reply:
yes this is used by the NetworkConnectionInterceptor class to log network states